kzz7yh-f124550/cod-ve09v2_kzz7yh-f124550.xml

```diff
@@ -115,7 +116,7 @@
     </p>
                <p xml:id="kzz7yh-f124550-d1e206">
                   <lb ed="#V" n="42"/>Ad primum in oppositum dicendum: quod illa 
-      au<lb ed="#V" n="43" break="no"/>ctoritas osee intelligenda est de 
+      au<lb ed="#V" n="43" break="no"/>ctoritas <!--word-->osee intelligenda est de 
```

Comment:
This is correct. The word is "osee" for "hosea" in the genitive form. It's a book of the bible :) 
```suggestion
      au<lb ed="#V" n="43" break="no"/>ctoritas osee intelligenda est de 
```